src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import Navbar from "../../components/NavBar/Navbar";
+import "./Home.css";
+import men from "../../assets/images/men.webp";
```

Comment:
Recently came to know about barrel exports . I think that would be a better practice

Reply:
I will do that once completed all functionalities